java/libphonenumber/src/com/google/i18n/phonenumbers/SingleFileMetadataSourceImpl.java

```diff
@@ -97,58 +94,25 @@ public PhoneMetadata getMetadataForNonGeographicalRegion(int countryCallingCode)
   void loadMetadataFromFile() {
     InputStream source = metadataLoader.loadMetadata(fileName);
     if (source == null) {
-      logger.log(Level.SEVERE, "missing metadata: " + fileName);
+      // This should not happen since clients shouldn't be using this implementation!
```

Comment:
I don't understand this comment. If they shouldn't be using it, why is the code here?